config/dev.yml

```diff
@@ -19,6 +19,7 @@ hint:
   oauth2_client_id: VAULT:secret/hint/oauth2/auth0:id
   oauth2_client_secret: VAULT:secret/hint/oauth2/auth0:secret
   oauth2_client_url: VAULT:secret/hint/oauth2/auth0:url
+  oauth2_login_method: true
```

Comment:
Bit pedantic but should we put this at the top of all the oauth configuration? Could also put this into a block like Rich suggested on a previous PR

```
authentication:
  use_oauth: true
  client_id: VAULT:secret/hint/oauth2/auth0:id
  client_secret: VAULT:secret/hint/oauth2/auth0:secret
  client_url: VAULT:secret/hint/oauth2/auth0:url
```